kpi/views/v2/asset_snapshot.py

```diff
@@ -79,8 +79,14 @@ def manifest(self, request, *args, **kwargs):
         """
         snapshot = self.get_object()
         asset = snapshot.asset
-        files = asset.asset_files.filter(file_type=AssetFile.FORM_MEDIA,
-                                         date_deleted__isnull=True)
+        form_media_files = list(
+            asset.asset_files.filter(
+                file_type=AssetFile.FORM_MEDIA,
+                date_deleted__isnull=True,
+            )
+        )
+        paired_data_files = list(PairedData.objects(asset).values())
+        files = form_media_files + paired_data_files
```

Comment:
in this context, why do the `PairedData` objects have to be treated separately and instantiated before being sent to the serializer, i.e. what's the difference between doing this special handling and changing `asset_files.filter()` to include both form media and paired data files?

Reply:
they're treated separately because `PairedData` implements the `OpenRosaManifestInterface` differently from `AssetFile`